chapters/planning-data-work.tex

```diff
@@ -1,147 +1,465 @@
-its structure is the same in each team member's computer.
-What may differ is the path to the project folder (the highest-level shared folder).
-This is reflected in the master script in such a way that
-the only change necessary to run the entire code from a new computer
-is to change the path to the project folder.
-The code in \texttt{stata-master-dofile.do} shows how folder structure is reflected in a master do-file.
+For our group, maintaining a single unified directory structure
+across the entire portfolio of projects means that everyone
+can easily move between projects without having to reorient
+themselves to how files and folders are organized.
+
+Our suggested file structure is not for everyone.
+But if you do not already have a standard file structure across projects,
+it is intended to be an easy template to start from.
+This system operates by creating a \texttt{DataWork} folder at the project level,
+and within that folder, it provides standardized directory structures
+for each data source (in the primary data context, ``rounds'' of data collection).
+For each, \texttt{iefolder} creates folders for raw encrypted data,
+raw deidentified data, cleaned data, final data, outputs, and documentation.
+In parallel, it creates folders for the code files
+that move the data through this progression,
+and for the files that manage final analytical work.
+The command also has some flexibility for the addition of
+folders for non-primary data sources, although this is less well developed.
+The \texttt{ietoolkit} package also includes the \texttt{iegitaddmd} command,
+which can place \texttt{README.md} placeholder files in your folders so that
+your folder structure can be shared using Git. Since these placeholder files are in
+\textbf{Markdown} they also provide an easy way
+to document the contents of every folder in the structure.
+  \index{Markdown}
+
+The \texttt{DataWork} folder may be created either inside
+an existing project-based folder structure, or it may be created separately.
+It's usually created by the leading RA in agreement with the PI.
+Increasingly, our recommendation is to create the \texttt{DataWork} folder
+separately from the project management materials,
+reserving the ``project folder'' for contracts, Terms of Reference, briefs and other administrative or management work.
+  \index{project folder}
+This is so the project folder can be maintained in a synced location like Dropbox,
+while the code folder can be maintained in a version-controlled location like GitHub.
+(Remember, a version-controlled folder \textit{should not}
+be stored in a synced folder that is shared with other people.
+Those two types of collaboration tools function very differently
+and will almost always create undesired functionality if combined.)
+Nearly all code files and raw outputs (not datasets) are best managed this way.
+This is because code files are usually \textbf{plaintext} files,
```

Comment:
```suggestion
This is because code files are always \textbf{plaintext} files,
```
Related to issue #192, but this is something that we might want to change anyways.